packages/design-system/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Accordion } from './Accordion';
+import { AccordionItem } from './AccordionItem';
+import { useArgs } from '@storybook/preview-api';
+
+const meta: Meta<typeof Accordion> = {
+  title: 'Components/Accordion',
+  component: Accordion,
+  args: {
+    bordered: true,
+    openItems: [0],
```

Comment:
I think this should be moved to the controlled story, because it only works there, and it's not a prop of the component.